docs-gen/content/ecosystem/tools.md

```diff
@@ -1,15 +1,16 @@
 ---
-title: "Tools"
-date: 2021-07-04T13:27:46+0000
+title: "Offcial VSS-Tools"
+date: 2023-05-18T13:27:46+0000
 weight: 1
 ---
 
-The VSS can be transformed into other formats or generated as code to integrate with other tooling environments and ecosystems.
+VSS can be transformed into other formats or generated as code to integrate with other tooling environments and ecosystems.
 
-Fig 1 shows the schematics of the top-level process.
+The COVESA [vss-tools project](https://github.com/covesa/vss-tools) contains various helpful converters and is also used to validate the syntactic validity of the VSS standard catalogue. 
+
+If new features are introduced to the VSS specification, the community will ensure they are supported in the official vss-tools as well.
 
 ![Signal tree](/vehicle_signal_specification/images/multi_target.png)<br>
-*Fig 1. Generating documents from specification*
+*Fig 1. Generating various output formats from a VSS specification*
```

Comment:
If you have time, modernizing/recreating this picture would be great. We do not have a markdown generator, and the names of the *.vspec files shown does not match "real files". Maybe better to make it more general, like showing one box for "Standard catalog" and one or more boxes for overlays.

Reply:
I will look into it